api/application/applications.go

```diff
@@ -8,139 +8,114 @@ import (
 )
 
 //WIP
-func Fetch(network *net.Network) string {
+func Fetch(network *net.Network) (string, error) {
 	url := "/v1/applications/fetch"
 	body, err := network.SendGetRequest(url)
 	if err != nil {
-		fmt.Println(err)
+		return string(body), err
```

Comment:
Need to be careful here, as it is likely that string var body will not have been created. Safer to just return empty string "".